components/03-modules/footer/_footer.scss

```diff
@@ -133,4 +134,23 @@ $footer__copywrite-padding-left\@large : $spacer--medium;
             padding-left: $footer__copywrite-padding-left\@large;
         }
     }
+
+    &__scroll-top {
+        position: absolute;
+        right: 0;
+        top: $footer__scroll-top-top;
+
+        &:hover,
+        &:focus {
+            .footer__icon {
+                fill: $footer__icon-fill-hover;
+            }
+        }
+    }
+
+    &__icon {
+        width: $footer__icon-sizing;
+        height: $footer__icon-sizing;
```

Comment:
icon-size seems better name ;) anyway what if in some project height would be different? I also used sometimes just "-size" but not sure if we shouldn't provide vars for width and height

Reply:
So far when you have child components based on alpaca when you want to update styles we are overwriting the file and we can change the variables and all other stuff here. Not necessary to setting 2 different variables

Reply:
and renamed to `size`